### PR TITLE
Update instructions for running benchmarks

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

**Comment by abarth** [comment]
TBR @eseidelGoogle

File: packages/unit/benchmark/README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-==========
+# Benchmarks
+
 This directory (and its sub-directories) contain benchmarks for Flutter.
 The reporting format for benchmarks is not standardized yet, so benchmarks
-here are typically run by hand. To run a particular benchmark, use a command
-similar to that used to run individual unit tests. For example:
+here are typically run by hand. To run a benchmark:
+
+1. Build `sky_shell` for Linux Release using the instructions in the
```
Comment (eseidelGoogle) [inline packages/unit/benchmark/README.md]: I'm surprised we haven't renamed it cocoon or something by now. :p

**Comment by eseidelGoogle** [comment]
lgtm

File: packages/unit/benchmark/README.md (hunk 1 above)
Comment (abarth) [inline packages/unit/benchmark/README.md]: Yeah, the binaries still need new names.

**Pull request merged**